tests/test_utils.py

```diff
@@ -236,7 +236,8 @@ def test_instantiate_adam() -> None:
 def test_instantiate_with_missing_module() -> None:
 
     with pytest.raises(
-        HydraException, match=re.escape(" No module named 'some_missing_module'")
+        InstantiateException,
+        match=re.escape("Error calling 'tests.ClassWithMissingModule'"),
```

Comment:
do we have another test that validates that the error message contains the name of the underlying missing module?

Reply:
Adding one now :)

Reply:
Ouch. This test is problematic. I left a bug in the code and the test didnt fail. Fixing both.